test/bdd/steps/network.js

```diff
@@ -466,6 +466,33 @@ Then(/^wallet and profile balances should diff by (\d+)$/, function (tokenDiff)
     expect(!!dc.state.newWalletBalance, 'newWalletBalance node not defined. Use other step to define it.').to.be.equal(true);
     expect(!!dc.state.oldWalletBalance, 'oldWalletBalance node not defined. Use other step to define it.').to.be.equal(true);
 
-    expect(dc.state.oldProfileBalance - dc.state.newProfileBalance, 'Profile diff should be equal to withdrawal amount').to.be.equal(tokenDiff);
-    expect(dc.state.newWalletBalance - dc.state.oldWalletBalance, 'Wallet diff should be equal to withdrawal amount').to.be.equal(tokenDiff);
+    expect(Math.abs(dc.state.oldProfileBalance - dc.state.newProfileBalance), 'Profile diff should be equal to withdrawal amount').to.be.equal(tokenDiff);
+    expect(Math.abs(dc.state.newWalletBalance - dc.state.oldWalletBalance), 'Wallet diff should be equal to withdrawal amount').to.be.equal(tokenDiff);
+});
```

Comment:
Provided values are rounded and should not be taken into formulas without precision param.